docs/installation.md

```diff
@@ -89,6 +91,11 @@ BENTO_GIT_NAME=David  # Change this to your name
 BENTO_GIT_EMAIL=do-not-reply@example.org  # Change this to your GitHub account email
 ```
 
+You should at least fill to the following settings, which are not set in the example file:
+* BENTOV2_SESSION_SECRET
+* BENTO_AUTH_DB_PASSWORD
+* BENTO_AUTHZ_DB_PASSWORD
+
```

Comment:
also need to set WES client secret, which comes from the output of init-auth - also maybe adding extra caution again that further changes are needed in production